exercises/practice/bob/create_test_table.sql

```diff
@@ -0,0 +1,48 @@
+DROP TABLE IF EXISTS tests;
+CREATE TABLE IF NOT EXISTS tests (
+    -- uuid and description are taken from the test.toml file
+    uuid TEXT PRIMARY KEY,
+    description TEXT NOT NULL,
+    -- The following section is needed by the online test-runner
+    status TEXT DEFAULT 'fail',
+    message TEXT,
+    output TEXT,
+    test_code TEXT,
+    task_id INTEGER DEFAULT NULL,
+    -- Here are columns for the actual tests
+    input INT NOT NULL,
+    expected TEXT NOT NULL
+);
+
+-- Note: the strings below contain literal tab, newline, carriage returns.
+
+INSERT INTO tests (uuid, description, input, expected)
+    VALUES
+        ("e162fead-606f-437a-a166-d051915cea8e","stating something","Tom-ay-to, tom-aaaah-to.","Whatever."),
+        ("73a966dc-8017-47d6-bb32-cf07d1a5fcd9","shouting","WATCH OUT!","Whoa, chill out!"),
+        ("d6c98afd-df35-4806-b55e-2c457c3ab748","shouting gibberish","FCECDFCAAB","Whoa, chill out!"),
+        ("8a2e771d-d6f1-4e3f-b6c6-b41495556e37","asking a question","Does this cryogenic chamber make me look fat?","Sure."),
+        ("81080c62-4e4d-4066-b30a-48d8d76920d9","asking a numeric question","You are, what, like 15?","Sure."),
+        ("2a02716d-685b-4e2e-a804-2adaf281c01e","asking gibberish","fffbbcbeab?","Sure."),
+        ("c02f9179-ab16-4aa7-a8dc-940145c385f7","talking forcefully","Hi there!","Whatever."),
+        ("153c0e25-9bb5-4ec5-966e-598463658bcd","using acronyms in regular speech","It's OK if you don't want to go work for NASA.","Whatever."),
+        ("a5193c61-4a92-4f68-93e2-f554eb385ec6","forceful question","WHAT'S GOING ON?","Calm down, I know what I'm doing!"),
+        ("a20e0c54-2224-4dde-8b10-bd2cdd4f61bc","shouting numbers","1, 2, 3 GO!","Whoa, chill out!"),
+        ("f7bc4b92-bdff-421e-a238-ae97f230ccac","no letters","1, 2, 3","Whatever."),
+        ("bb0011c5-cd52-4a5b-8bfb-a87b6283b0e2","question with no letters","4?","Sure."),
+        ("496143c8-1c31-4c01-8a08-88427af85c66","shouting with special characters","ZOMG THE %^*@#$(*^ ZOMBIES ARE COMING!!11!!1!","Whoa, chill out!"),
+        ("e6793c1c-43bd-4b8d-bc11-499aea73925f","shouting with no exclamation mark","I HATE THE DENTIST","Whoa, chill out!"),
+        ("aa8097cc-c548-4951-8856-14a404dd236a","statement containing question mark","Ending with ? means a question.","Whatever."),
+        ("9bfc677d-ea3a-45f2-be44-35bc8fa3753e","non-letters with question",":) ?","Sure."),
+        ("8608c508-f7de-4b17-985b-811878b3cf45","prattling on","Wait! Hang on. Are you going to be OK?","Sure."),
+        ("bc39f7c6-f543-41be-9a43-fd1c2f753fc0","silence","","Fine. Be that way!"),
```

Comment:
There are no spaces between values. `","` ==> `", "`

Reply:
Yep, that's how jq emits CSV.

Reply:
BUT, those double quotes need to be all single quotes. I'll have to update that file.

Reply:
The test generator in raindrops cleans that up. If we have tooling that does that cleaner, it would be nice to reuse that existing tooling.